src/content/changes/highlights/2024-07-21.md

```diff
@@ -0,0 +1,186 @@
+connectivity/compare/v0.5.4...v0.6.0)
+        - Update kubectl container image to version v1.26.0 for WorkloadCluster Ip Job
+        - Increase pod and container SecurityContext settings for WorkloadCluster Ip Job
+        - Execute `kubectl apply` with `--server-side=true --field-manager='kubectl-client-side-apply' --force-conflicts` flags in WorkloadCluster Ip Job
+        - Remove unused tests under helm directory.
+- [security-bundle](https://github.com/giantswarm/security-bundle)
+    - [1.8.0](https://github.com/giantswarm/security-bundle/compare/v1.7.1...v1.8.0)
+        - Add `kyverno-crds` app to handle Kyverno CRD install.
+        - Update `kyverno` (app) to v0.17.15. This version disables the CRD install job in favor of `kyverno-crds` App.
+- [kyverno-app](https://github.com/giantswarm/kyverno-app)
+    - [0.17.15](https://github.com/giantswarm/kyverno-
+app/compare/v0.17.14...v0.17.15)
+        - Set VPA max 6 CPU / 24Gi memory and adjust default requests/limits for `reports-controller`.
+        - Set VPA max 4 CPU / 8Gi memory and adjust default requests/limits for `background-controller`.
+        - Set starting CPU limit of request+25% for `cleanup-controller`.
+        - Disable Kyverno CRDs install Job in favor of `kyverno-crds` App.
+- [kyverno-crds](https://github.com/giantswarm/kyverno-crds)
+      - [1.11.1](https://github.com/giantswarm/kyverno-crds/compare/v1.11.0...v1.11.1)
+        - Removed unpopulated labels and fixed the team label.
+      - [1.11.0](https://github.com/giantswarm/kyverno-crds/releases/tag/v1.11.0)
+          - First release of the Kyverno CRDs App for Kyverno v1.11.
+
+- [chart-operator](https://github.com/giantswarm/chart-operator)
+    - [3.3.0](https://github.com/giantswarm/chart-operator/compare/v3.2.1...v3.3.0)
+        - Added and enabled by default an `emptyDir` volume under `tmp` of the deployment for artifact pulls so that we can make the rest of the root file system read only under the security context.
+        - Added control over the VPA max allowed resources.
+        - Requests and limits are always set on the deployment.
+        - Increased default requests and limits because of the introduction of the `emptyDir` volume under `/tmp`.
+        - Enabled read only root file system by default on the security context.
+        - Update PolicyExceptions to v2 and fallback to v2beta1.
+- [cluster-apps-operator](https://github.com/giantswarm/cluster-apps-operator)
+    - [2.23.0](https://github.com/giantswarm/cluster-apps-
+operator/compare/v2.22.1...v2.23.0)
+        - Bump `chart-operator` to version [v3.3.0](https://github.com/giantswarm/chart-operator/releases/tag/v3.3.0)
+- [zot](https://github.com/giantswarm/zot)
+    - [1.1.0](https://github.com/giantswarm/zot/compare/v1.0.0...v1.1.0)
+        - Update to `project-zot/helm-charts` version `0.1.58`.
+        - Update Default Zot container image tag to version [v2.1.0](https://github.com/project-zot/zot/releases/tag/v2.1.0)
```

Comment:
@weatherhog These are from Honeybadger. Please adapt the content to the standard format (name, version, description)
[example](https://raw.githubusercontent.com/giantswarm/docs/main/src/content/changes/highlights/2024-02-15.md)

Reply:
removed the honeybagder highlights as these do not need to be announced to our customers